### ISSUE TITLE
berry crashes when xev started

### ISSUE THREAD

**Issue by TaylanTatli** [JLErvin/berry#43]
Here is the output:
```
berry: font not specified, loading default font
berry: Successfully opened display
berry: Successfully assigned atoms
berry: Successfully set initial properties
berry: Setting up monitors
berry: Found 1 screens active
berry: Screen #0 with dim: x=0 y=0 w=1366 h=768
berry: Successfully setup monitors
berry: Allocating color values
berry: Receieved new 16 event
berry: Receieved new 33 event
berry: Handling 33 event
berry: Recieved event from berryc
berry: Receieved new 33 event
berry: Handling 33 event
**!!!!!!!!!!Same outputs so I deleted them!!!!!!!!!**
berry: Text height is 4
berry: Drawing text on client
berry: Receieved new 28 event
berry: Handling 28 event
berry: Handling property notify event
berry: Receieved new 28 event
berry: Handling 28 event
berry: Handling property notify event
berry: Updating client title
berry: Text height is 7
berry: Drawing text on client
berry: Receieved new 16 event
berry: Receieved new 20 event
berry: Handling 20 event
berry: Handling map request event
berry: client has class @?�ߒU
```
When i run xev -event keyboard berry gives berry: client has class @?�ߒU output and crashes.

Here is my xinitrc
```
# Source xinitrc.d scripts
if [ -d /etc/X11/xinit/xinitrc.d ] ; then
    for f in /etc/X11/xinit/xinitrc.d/?*.sh ; do
        [ -x "$f" ] && . "$f"
    done
    unset f
fi

# load .Xresources
/usr/bin/xrdb -merge $HOME/.Xresources

# Set wallpaper
nitrogen --restore &


# Compton
compton --config ~/.config/compton.conf &

# Start PulseAudio
start-pulseaudio-x11 &

# Polkit
/usr/lib/polkit-gnome/polkit-gnome-authentication-agent-1 &

# Start Unclutter
unclutter &

# Fonts
xset +fp ~/.local/share/fonts/Bitmap
xset fp rehash

# cursor
xsetroot -cursor_name left_ptr

# numlock
numlockx on &

# Tray apps
volumeicon &
cbatticon &
nm-applet &

# Start wm
sxhkd &
exec berry >~/.xsession-errors 2>&1
```
 Berry is installed from AUR and version is 0.1.2.r7.g8aa8085
Also berry is not starting without chmod +x autostart
And berry crashes a lot when starting apps other than xev.
Thanks.

**Comment by JLErvin** [comment]
@TaylanTatli thanks for the detailed writeup - I was able to reproduce this on my end. Should be a quick fix but I'll keep you updated.

**Comment by JLErvin** [comment]
@TaylanTatli should be fixed as of 67f16df94dfcb8da70790085fd8390eabe734f6f on the development branch. I will continue testing for a day or two before merging to master but feel free to build from there and see if this fixes your issue.

**Comment by JLErvin** [comment]
Merged with master as of 0.1.3

**Comment by TaylanTatli** [comment]
It works, thank you.
